run-build-functions.sh

```diff
@@ -177,10 +162,29 @@ run_yarn() {
   export PATH=$(yarn bin):$PATH
 }
 
-run_npm_set_temp() {
```

Comment:
This is not used anymore: 

![CleanShot 2022-10-12 at 12 48 09](https://user-images.githubusercontent.com/11156362/195323703-e5771854-b63a-40c9-83c1-b3ac6e15e85d.png)